### PR TITLE
Feat: Create README.md for invite service

### PR DESCRIPTION
**Pull request by skelet0no**
Created README.md for invite service

### CODE REVIEW

**Review by ZixMai — APPROVED** [review]

File: invite_service/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+___
+# Invite service
+[![Build invite service](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_invite_service.yaml/badge.svg)](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_invite_service.yaml)
+___
+**Установка и описание**
+___
+1. **Образец .env файла**
+	```
+	DATABASE_URL="DATABASE_URL"
```
Comment (Nhsdkk) [inline invite_service/README.md:10]: Add comments for variables

File: invite_service/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+       - Установка осуществляется через **docker**, для установки пакета микросервиса введите и выполните команду:
+		```
+    	docker pull ghcr.io/unspokenteam/invite_service:latest
+ 		```
+       - Создайте .env файл, укажите его путь в команде и выполните её:
+		```
+		docker run -d -p 8082=8082 —env-file $PATH_TO_ENV ghcr.io/unspokenteam/invite_service:latest
+ 		```
+
+	2. **Через poetry**
```
Comment (Nhsdkk) [inline invite_service/README.md:36]: Add proto and prisma client generate steps

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

File: invite_service/README.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+[![Build invite service](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_invite_service.yaml/badge.svg)](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_invite_service.yaml)
+___
+**Установка и описание**
+___
+1. **Образец .env файла**
+	```
+	DATABASE_URL="DATABASE_URL"
+	ENVIRONMENT="ENVIRONMENT"
+	```
+ 	- где:
```
Comment (Nhsdkk) [inline invite_service/README.md]: Move this to comments inside dotenv example

File: invite_service/README.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      - DATABASE_URL - адрес базы данных
+      - ENVIRONMENT - название окружения
+___
+2. **Установка**:
```
Comment (Nhsdkk) [inline invite_service/README.md:15]: Add pre-installation step to create db

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

**Review by MaxPlays35 — APPROVED** [review]

File: invite_service/README.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+___  
+**Перед установкой**:
+
+1. Сделать ```docker pull``` для custom_postgres или подготовить собственный instance PostgreSQL
+```bash
+docker pull ghcr.io/unspokenteam/custom_postgres:latest
+```
+2. Подготовить локальный инстанс redis ([Setup redis with docker](https://redis.io/docs/latest/operate/oss_and_stack/install/install-stack/docker/))
```
Comment (Nhsdkk) [inline invite_service/README.md]: Delete this step

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

**Review by Nhsdkk — APPROVED** [review]
LGTM

**Pull request merged**